src/main/resources/templates/fragments.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+<!--header fragment-->
+<head th:fragment="header">
+    <head>
+        <meta charset="UTF-8">
+    </head>
+    <link href="/css/reset.css" rel="stylesheet">
+    <link href="/css/base.css" rel="stylesheet">
+    <link href="/css/layout.css" rel="stylesheet">
+    <link href="/css/module.css" rel="stylesheet">
+    <link href="/css/state.css" rel="stylesheet">
+</head>
+
+
+<!--New Entry Fragment-->
+<aside th:fragment="newEntry" id="newEntry">
```

Comment:
Today I learned "aside" is a tag.  It's probably not exactly the correct one for this, though.